cs/game.php

```diff
@@ -0,0 +1,174 @@
+    'finesse'=> 'Obratnost',
+    'save'=> 'Uložit',
+    'createRoom'=> 'Vytvořit místnost',
+    'editRoom'=> 'Upravit',
+    'refreshList'=> 'Obnovit',
+    'roomName'=> 'Název místnosti',
+    'playersCount'=> 'Hráčů',
+    'usePreset' => 'Použít přednastavení',    //preset = predefined room settings
+    'default' => 'Výchozí',
+    'defaultDesc' => 'Výchozí nastavení',
+    'otherOptions' => 'Další možnosti',
+    'customPreset' => 'Použít vlastní přednastavení',
+    'browsePresets' => 'Procházet přednastavení',
+    'submitPreset' => 'Vytvořit nové',
+    'savePreset' => 'Uložit přednastavení',
+    'customRooms'=> 'Vlastní místnosti',
+    'overflowRooms'=> 'Přeplněné místnosti',
+    'spectateRooms'=> 'Místnosti jen k dívání',
+    'gmode'=> 'Režim',
+    'gmodeStandard'=> 'Standardní',
+    'gmodeCheese'=> 'Cheese',
+    'gmodeTeam'=> 'Tým',
+    'isPrivate'=> 'Soukromý',
+    'simple'=> 'Jednoduchý',
+    'advanced'=> 'Pokročilý',
+    'moreSett'=> 'Další nastavení',
+    'attack'=> 'Útok',
+    'gDist'=> 'Rozdělení garbage',
+    'create'=> 'Vytvořit',
+    'applyCh'=> 'Použít změny',
+    'cancel'=> 'Zrušit',
+    'about' => 'O hře',
+    'donate' => 'Přispět',
+    'hdThread' => 'Vlákno na HardDrop',
+    'chatSend' => 'Odeslat',
+    'rememberName' => 'Zapamatovat přezdívku',
+    'survival' => 'Přežití',
+    'mapDownstack' => 'Obrázky',
+    'ultra' => 'Ultra',
+    '20TSD' => '20TSD',
+    'all' => 'Vše',
+    'preset' => 'Přednastavení',
+    'save' => 'Uložit',
+    'audioCredits' => 'Zvuky od',
+    'fullScreen' => 'Celá obrazovka',
+    'showHoldQueue' => 'Zobrazit frontu a uschovaný dílek',
+    'showStats' => 'Zobrazit statistiky',
+    'roomLimits' => 'Omezit připojení k místnosti',
+    'setLimitsInfo' => 'Můžete nastavit požadavky na hráče, kteří se chtějí připojit. Pokud žádné požadavky nechcete, nechte políčko prázdné',
+    'addLimit' => 'Přidat limit pro připojení',
+    's40L' => 'Sprint 40 řad',
+    'accGametime' => 'Uplynulý čas',
+    'applyClose' => 'Uložit a zavřít',
+    'teamSel' => 'Výběr týmu',
+    'myTeam' => 'Hra trvá už',
+    'seconds' => 'sekund',
+    'minutes' => 'minut',
+    'hours' => 'hodin',
+    'ms' => 'ms',
+    's' => 's',
+    'hrs' => 'h',
+
+    //all room settings
+    'attackTable' => 'Tabulka útoků',
+    'comboTable' => 'Kombo útok',
+    'gBlocking' => 'Přidat řádky (garbage)',
+    'blocksType' => 'Druhy bloků',
+    'randomizer' => 'Náhoda',
+    'previews' => 'Náhledy',
+    'solidGarbage' => 'Upevnit garbage',
+    'solidAfterSecs' => 'za',
+    'lockDelay' => 'Prodleva před upevněním',
+    'clearDelay' => 'Prodleva před vyčištěním',
+    'speedLimit' => 'Omezení rychlosti',
+    'gravityLvl' => 'Síla gravitace',
+    'garbageDelay' => 'Prodleva řádků garbage',
+    'garbageMess' => 'Složitost řádků garbage',
+    'solidAtk' => 'Útok pevnými řádky',
+    'noFW' => 'Zakázat šířku 4',
+    'hostStart' => 'Start správce',
+
+    'translationBy' => 'Přeložil',
```

Comment:
This is meant as translation author, it will be shown in the list of translators.
```suggestion
    'translationBy' => 'isametry',
```